src/main/java/com/jcraft/jsch/Slf4jLogger.java

```diff
@@ -1,12 +1,26 @@
 package com.jcraft.jsch;
 
+import org.slf4j.Logger;
```

Comment:
I wonder if it would be better to drop this import statement and just always fully qualify the package for the `Logger` references below so that it is easier to reason about which type of `Logger` is being referred to?

Reply:
I think, the class is simple enough that one can be done by import and the other fully qualified. IMHO the fully qualified way is less readable.

Reply:
Well, it does lead to a bit of inconsistencies between various implementations, as some of the now have an import statement for the logging framework's `Logger` class and some don't.

We should make it consistent between them all whether they do or don't import it.

Reply:
I've checked the loggers and haven't found any inconsistencies